src/api/endpoints/interLexURIStructureAPI.ts

```diff
@@ -483,16 +483,32 @@ export const useGetOpsUserLogin = <TData = Awaited<ReturnType<typeof getOpsUserL
 
 
 
-export const postOpsUserLogin = (
+// export const postOpsUserLogin = (
     
- options?: SecondParameter<typeof customInstance>,) => {
+//  options?: SecondParameter<typeof customInstance>,) => {
       
       
-      return customInstance<void>(
-      {url: `https://uri.olympiangods.org/u/ops/user-login`, method: 'POST'
-    },
-      options);
-    }
+//       return customInstance<void>(
+//       {url: `https://uri.olympiangods.org/u/ops/user-login`, method: 'POST'
+//     },
+//       options);
+//     }
+export const postOpsUserLogin = (
+  userData: { username: string; password: string },
+  options?: SecondParameter<typeof customInstance>
+) => {
+  return customInstance<void>(
+    {
+      url: `https://uri.olympiangods.org/u/ops/user-login`,
```

Comment:
use the const variables instead of hardcoding the url and endpoints please, so that if later we need to change these we can change them from a single point.